.github/renovate.json

```
@@ -14,7 +14,7 @@
      "stabilityDays":0
   },
   "separateMajorMinor":false,
-  "ignorePaths": ["infra/concourse/**", "config-connector/**", "docs/**"],
+  "ignorePaths": ["config-connector/**", "docs/**"],
```

since we removed docs too
```suggestion
  "ignorePaths": ["config-connector/**"],
```